### DISCUSSION TITLE
Force script execution order

### DISCUSSION THREAD

**Discussion by cobaltt7** [ScratchAddons/ScratchAddons#1206] [Help] [answered]
For my work-in-progress addon, I have two scripts defined in addon.json: one that executes on all pages and one that only executes on some.

How do I force the script that executes on all pages to finish executing before the other starts? Do I just have to define it first in addon.json? Or is there more that I must do?

**Comment by WorldLanguages** [comment, marked as answer]
It depends on the order you specified in the userscripts array in addon.json
